css/properties/mask-size.json

```diff
@@ -33,10 +33,12 @@
               "version_added": null
             },
             "safari": {
-              "version_added": false
+              "version_added": "4",
+              "prefix": "-webkit"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "3",
```

Comment:
Where do you have this from? I don't see it in the doc?

Reply:
https://docs.google.com/document/d/1GBlFCgDm3AWI10IlOFoTRhXBbaIvcxqZhMmScf7KJcA/edit#bookmark=id.1664s55

Reply:
Actually, the Safari iOS version is wrong anyways.  Whoops!